src/crate/testing/layer.py

```diff
@@ -44,7 +44,7 @@
 
 CRATE_CONFIG_ERROR = 'crate_config must point to a folder or to a file named "crate.yml"'
 HTTP_ADDRESS_RE = re.compile(
-    '.*\[http +\] \[.*\] .*'
+    '.*\[i.c.p.h.CrateNettyHttpServerTransport] \[.*\] .*'
```

Comment:
Should we keep compat here for the older format? If not it's kind of a breaking changes for the testing layer.

Reply:
Yes, good point

Reply:
I agree, we should keep it compatible.